stable/rancher/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+  - jgreat
+  - ibuildthecloud
+  - cloudnautique
+  - deniseschannon
+  - vincent99
```

Comment:
All approvers need to be members of the kubernetes org on github. Please see:  https://github.com/kubernetes/community/blob/master/community-membership.md

Reply:
I guess I'm uncertain on how we would achieve this. We (Rancher Labs) would really like to have a chart to install Rancher in the main public helm repo, manage and maintain this chart.

How do other companies handle this?

Reply:
I am not sure. @unguiculus might have a better answer